vault/logical_system.go

```diff
@@ -1098,12 +1102,17 @@ func (b *SystemBackend) handlePluginCatalogUpdate(ctx context.Context, req *logi
 		return logical.ErrorResponse("missing command value"), nil
 	}
 
+	args := d.Get("args").([]string)
+	// For backwards compatibility, also accept args as part of command.
+	parts := strings.Split(command, " ")
+	args = append(parts[1:], args...)
```

Comment:
Please do a length check before this line to make sure the command actually has more than one value...otherwise it'll panic.

Reply:
Length >0 is assumed by the check for the empty string above (as with the previous iteration of the code).  But I'll add it anyway :-)